conf/mini_provisioner/v2/setup.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
rename file to ha_setup.yaml.

Reply:
Added : Placeholder for common configuration files in v1 and v2.